services/src/api/handlers/layers.rs

```diff
@@ -314,6 +334,218 @@ async fn list_collection_handler<C: ApplicationContext>(
     Ok(web::Json(collection))
 }
 
+// Returns the search capabilities of the given provider
+#[utoipa::path(
+    tag = "Layers",
+    get,
+    path = "/layers/collections/search/capabilities/{provider}",
+    responses(
+        (status = 200, description = "OK", body = SearchCapabilities,
```

Comment:
I guess it would make sense to make the capabilities more generic, and e.g. include the ability to list collections and in the future more

Reply:
[e336e65](https://github.com/geo-engine/geoengine/pull/908/commits/e336e6508a1ab96cc4620a5e91c8a2731dbaffe6)

Reply:
the search capabilites for the "Data catalogue" (previously "Layer Collections") now returns no search capabilites and thus can no longer searched in the UI.

Reply:
```Rust
#[async_trait]
impl<Tls> LayerCollectionProvider for PostgresDb<Tls>
where
    Tls: MakeTlsConnect<Socket> + Clone + Send + Sync + 'static,
    <Tls as MakeTlsConnect<Socket>>::Stream: Send + Sync,
    <Tls as MakeTlsConnect<Socket>>::TlsConnect: Send,
    <<Tls as MakeTlsConnect<Socket>>::TlsConnect as TlsConnect<Socket>>::Future: Send,
{
    fn capabilities(&self) -> ProviderCapabilities {
        ProviderCapabilities {
            listing: true,
            search: SearchCapabilities {
                search_types: SearchTypes {
                    fulltext: true,
                    prefix: true,
                },
                autocomplete: true,
                filters: None,
            },
        }
    }
```

This is not true. It returns search capabilities.
If you mean the layerDb in `pro`, this will be implemented in another PR and also did not work before.

Reply:
[54e9d61](https://github.com/geo-engine/geoengine/pull/908/commits/54e9d610d94c8e69131f7b613668c5da39543b81)